docs/docs/community/release-process.md

```diff
@@ -0,0 +1,68 @@
+---
+id: release-process-branching-strategy
```

Comment:
since I cannot leave comments on files names - this needs to be under `docs/community/maintainers`